BitwardenShared/UI/Platform/Settings/Settings/Other/OtherSettingsState.swift

```diff
@@ -19,4 +21,11 @@ struct OtherSettingsState {
 
     /// A toast message to show in the view.
     var toast: Toast?
+
+    // MARK: Computed Properties
+
+    /// Whether the connect to watch toggle should be shown.
+    var shouldShowConnectToWatchToggle: Bool {
+        UIDevice.current.model == "iPhone"
+    }
```

Comment:
🎨 Even though this is fine and achieves the goal it's not easy to mock and test. So in order to improve this and be consistent to how we implement this kind of stuff I'd make this a normal property:
```suggestion
    /// Whether the connect to watch toggle should be shown.
    var shouldShowConnectToWatchToggle: Bool = false
```
Then I'd inject in the `OtherSettingsProcessor` the `SystemDevice` object (this can be done by adding `& HasSystemDevice` to the `Services` typealias in there). And then in `loadInitialValues()` I would update the state with that:
```swift
state.shouldShowConnectToWatchToggle = services.systemDevice.isIPhone
```
As you can see I've used a property `isIPhone` which should be added to the `SystemDevice` extension so the logic can be reused and have everything concentrated in one place thus it's more maintainable:
```swift
var isIPhone: Bool {
    model == "iPhone"
}
```
🎨  And then add the tests for all these new logic which is easier now that the `SystemDevice` protocol is being used so in the tests you can use `MockSystemDevice` to change the `model` and check each behavior.